server/schedulers/adjacent_region.go

```diff
@@ -300,7 +300,7 @@ func (l *balanceAdjacentRegionScheduler) disperseLeader(cluster opt.Cluster, bef
 	}
 	op, err := operator.CreateTransferLeaderOperator("balance-adjacent-leader", cluster, before, before.GetLeader().GetStoreId(), target.GetID(), operator.OpAdjacent)
 	if err != nil {
-		log.Debug("fail to create transfer leader operator", zap.Error(err))
+		log.Debug("failed", zap.Error(errs.ErrCreateOperator.FastGenByArgs("transfer leader")))
```

Comment:
Does the error already handle in the `operator` package? maybe we only need to log it directly. cc @ZenoTan @rleungx

Reply:
If err is already a RFC error, we can just use it

Reply:
done